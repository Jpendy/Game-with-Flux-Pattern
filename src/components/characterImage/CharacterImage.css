.CharacterDead {
animation-name: dead;
animation-duration: 1.5s;
}

@keyframes dead {
    from{ transform: translateY(0px);}
    to{transform: translateY(500px);}
}