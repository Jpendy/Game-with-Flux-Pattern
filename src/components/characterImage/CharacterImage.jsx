import React from 'react';
import PropTypes from 'prop-types';

export default function CharacterImage({ image }) {
  return (   
    <figure>
      <img style={{ height: '300px' }} src={image} />
    </figure>
  );
}

CharacterImage.propTypes = {
  image: PropTypes.string.isRequired
};
