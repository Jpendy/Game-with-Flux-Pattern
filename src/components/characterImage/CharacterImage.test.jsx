import React from 'react';
import { shallow } from 'enzyme';
import CharacterImage from './CharacterImage';

describe('CharacterImage component', () => {
  it('renders CharacterImage', () => {
    const wrapper = shallow(<CharacterImage image={'image'} />);
    expect(wrapper).toMatchSnapshot();
  });
});
  
