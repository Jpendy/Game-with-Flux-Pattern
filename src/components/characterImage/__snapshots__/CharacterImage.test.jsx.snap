// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CharacterImage component renders CharacterImage 1`] = `
<figure>
  <img
    src="image"
    style={
      Object {
        "height": "300px",
      }
    }
  />
</figure>
`;
